Websites/bugs.webkit.org/PrettyPatch/PrettyPatch.rb

```diff
@@ -32,6 +32,7 @@ def self.prettify(string)
 
         $last_prettify_file_count = fileDiffs.length
         str << fileDiffs.collect{ |diff| diff.to_html }.join
+        str << '<script src="code-review.js?version=48"></script>'
```

Comment:
Moving this to the very last line of <body> so all elements are availabe by this time. This is a difference between .on() and .live(). Also using 'document' instead of 'body' didn't seems to work.